taxi/views.py

```diff
@@ -87,3 +102,18 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    fields = "__all__"
```

Comment:
In `DriverCreateView`, using `fields = "__all__"` can expose sensitive fields like passwords. Consider using a specific form class to handle driver creation securely.